db/seeds.rb

```diff
@@ -22,41 +22,41 @@ def create_categories
 
   def create_stickers
     @nodejs_sticker = @javascript_category.stickers.create(title: "Nodejs",
-                                         image_url: "http://devstickers.com/assets/img/cat/nodejs.png",
+                                         image: "http://devstickers.com/assets/img/cat/nodejs.png",
```

Comment:
Line is too long. [99/80]<br>Align the elements of a hash literal if they span more than one line.